### ISSUE TITLE
Review of 9.2 Case Study: Data Access Services

### ISSUE THREAD

**Issue by r-owen** [lsst-dm/spie-2018-dm-practices#37]
Consider "the build will fail" in: "and the build will be failed if these do not pass"

Regarding "some parts of the service suite have coverage by small- scale automated integration tests":
Remove the space after the hyphen and consider removing "coverage by"

Regarding a different part of that same sentence: "automated integration tests; these tests are run by Travis"
This implies that only the integrate tests are run by Travis. Whether that is true or not, I suggest rewording it to make it explicit which test are run by Travis. Breaking it into two sentences would probably help.

This wording seems a bit clumsy to me: "These deployments host a combination of synthetic test data..."
Consider shortening to: "These serve a combination of synthetic test data..."

Regarding: "Cluster deployments of the Data Access Services containers are done at present with a collection of ad-hoc administration scripts, but the project is working currently to replace those"
This seems convoluted; consider, instead: "Data Access Services containers are presently deployed using ad-hoc administration scripts, but the project is working to replace those..."

Regarding: "The Data Access services have been incubated along with the rest of the LSST software stack, sharing common CI infrastructure, software package management, build tooling, and release cadences.".
I find "have been incubated" confusing. I believe the point is that data access services were *built* sharing common CI infrastructure, etc.

Regarding: "it seems likely that periods of heavy activity on the data production and data service codes will phase and there will be a desire for decoupled release cadences for these."
In find "will phase" confusing. Do you mean "will likely be out of phase"?

9.3 Case Study: Deploying releases to JupyterLab

Consider removing "already" from: "The LSST Stack is already built within the context of a CI framework"

Please reword for clarity: "As our exploration of JupyterLab as the interactive notebook component of the LSST Science Platform42 proceeds, we have realized the need for automated deployments of the lab environment, which is simply a Kubernetes cluster and namespace."

Regarding the second paragraph (which starts with the sentence just quoted), perhaps I am just showing my ignorance here, but I thought first paragraph was all about automatic deployment of JupyterLab environment. For instance the first paragraph ends "From this we can construct a menu of the most recent daily, weekly, and release builds, and present a choice of stack versions to the user at JupyterHub login time.", which suggests that the tooling automatically makes these builds available.
In any case I probably will not be the only reader confused by this. Can you please try to clarify the difference between whatever the first paragraph is describing and the automatic deployment described by the second paragraph?

9.4. Deployment at IN2P3
Consider shortening "contributing to the annual LSST data release processing during the operations phase" to "contributing to the processing of the annual LSST data release". When else would this occur but during operations? At the very least consider shortening "during the operations phase" to "during operations".

I find "since 2015" a distraction that makes the sentence clumsy in: "since 2015 IN2P3 computing center (CC-IN2P3) has been consistently deploying releases".
Consider something like: "IN2P3 computing center (CC-IN2P3) deploys each release of the LSST software stack as soon as is is available". You can also note that this has been going on since 2015 but I don't think that adds much.

Regarding: "Two mechanisms have been used for this continuous deployment"
I strongly suggest using present tense for this; it's less convoluted. Also consider ditching the word "continuous".
Consider something like: "Two mechanisms are used for this deployment"

This seems a bit verbose: "We adopted a deployment policy which has proven effective over the period we have been performing this process. Each deployment, be it of a weekly or a stable release, is as self-contained as possible."
You say it is helpful but you don't say how and you have a very nice discussion of that topic later on. Consider deleting the first sentence. Also consider deleting "be it of a weekly or a stable release"; I don't think it adds information.

Run-on sentence; I suggest a period instead of a colon: "visualisation and rapid prototyping: they can easily choose the version"

**Comment by womullan** [comment]
9.2. was considerably shortened removing many of the problems alredy in my PR (I took in the failed->fail)
9.3. removed kubi namespace indeed it is not a good sentence and you are correct it was badly ordered I moved some text up to make it more logical 
9.4. thanks for the intro sentences clean up added - the rest again was already heavily modified.
